vector/src/main/res/drawable/reaction_rounded_rect_shape_off.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <size android:width="40dp" android:height="22dp"/>
```

Comment:
why we have a size here? And not in `reaction_rounded_rect_shape.xml`?

Reply:
I have no idea. It is a copy of `rounded_rect_shape_off.xml`. I will remove and test it.

Reply:
Fixed.